code/api/src/migrations/11-subscriptiondate.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
Great, we'll need to access this data - we might need your help figuring out exactly how we do that